jujugui/static/gui/src/app/components/machine-view/machine-view.js

```diff
@@ -154,21 +156,26 @@ YUI.add('machine-view', function() {
         var icon;
         var content;
         if (props.services.size() === 0) {
-          icon = 'add_16';
           content = (
-            <span className="link"
-              onClick={this._openStore}>
-              Add applications to get started
-            </span>
+            <div>
+              <p>
+                Drag and drop unplaced units to customise your deployment.
```

Comment:
This message is confusing when you have no units added. It should probably only be visible when there are unplaced units.